diffsync/__init__.py

```diff
@@ -71,6 +72,20 @@ class DiffSyncModel(BaseModel):
     Note: inclusion in `_attributes` is mutually exclusive from inclusion in `_identifiers`; a field cannot be in both!
     """
 
+    _enabled_attributes: ClassVar[Mapping[str, bool]] = defaultdict(lambda: True)
+    """Optional: dict of `{field_name: enabled}` entries detailling whether an attribute is enabled or not.
+    
+    When a field is not included in this dictionary, it means it is enabled (this also provides backwards
+    compatibility). When a field is
+    - included as `_enabled_attributes[field_name] == False`, this means that while the attribute has to defined like
+      any other, it will not take part in any diffsync logic
```

Comment:
Hmm. Probably I'm missing something, but can't the same effect be achieved with current functionality by simply overriding `_attributes` on the concrete subclass?

Reply:
I see it like @glennmatthews 🤔 , maybe we need more context?

Reply:
It appears you are correct and I have gone full circle 🤦🏻